DoSomething/gambit-conversations
# Transition cleanup

#### What's this PR do?

Now that all published default topic triggers have had their `response` reference field updated with transition entries - we can complete the TODO's of removing the interim code that handled referencing a topic entry.

Also refactors to deprecate the `template` helper, we no longer need the `isAskContinue` or `isTopicTemplate` functions per changes made in #441.

#### How should this be reviewed?
* Verify a `POST messages?origin=subscriptionStatusActive` request creates expected outbound message text and template
* Verify a trigger for a closed campaign sends closed campaign template, e.g. text `mascot`

#### Any background context you want to provide?

Holding off on consolidating the replies/template/macro configs to keep this reviewable (and knock out the rest of the photo post middleware)


#### Checklist
